src/addItem.js

```diff
@@ -0,0 +1,28 @@
+const addLeaderboardItems = async () => {
+  const score = document.getElementById('scoreInput');
+  const name = document.getElementById('nameInput');
+  const error = document.getElementById('ErrorMessage');
+
+  if (name.value.length && score.value.length) {
```

Comment:
[OPTIONAL] I think you might want to do better validations for the name and the score, such as formatting the name or making sure the user enters a number in a valid range.